scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -62,13 +68,19 @@ fi
 bundle_id=${1:-$HOSTNAME}
 dest_dir=${2:-$DEFAULT_DEST_DIR}
 
-while getopts ":b:t:c:s:" flag
+while getopts ":b:t:c:s:d:e:m:o:k:" flag
```

Comment:
I believe o<core_dump> and k<stack_trace> would be Boolean option (without parameters), so don't use :(colon) here. Otherwise it would expect some arguments here. If we are planning to specify some arguments as well then give some example as you have given to <no_of_days> and <type_of_log>

instead of -k, I think -t is better suitable option for traces.

Reply:
incorporated